src/java/com/sarality/app/view/dialog/CustomViewDialogFragment.java

```diff
@@ -16,22 +16,30 @@
 /**
  * A DialogFragment that render a custom view inside a Dialog.
  *
+ * TODO(abhideep): Add Action support.
+ *
  * @author abhideep@ (Abhideep Singh)
  */
-public abstract class CustomViewDialogFragment<V extends View, T> extends BaseDialogFragment implements DataConsumer<T> {
+public abstract class CustomViewDialogFragment<V extends View, T> extends DialogFragment implements DataConsumer<T> {
```

Comment:
The Actions can't be reused for CustomDialogFragment anyway. A CustomDialogFragment is simply a Fragment with a View that shows up in a Dialog. It has none of the traditional functionality associated with a simple AlertDialog. Hence there is no reuse of Actions. 

We don't have a Custom Dialog as yet, but this was added just in case we need one for the future, we don't go around changing the normal Initializer for AlertDialogFragment